packages/dma-contracts/tasks/verify-deployment/index.ts

```diff
@@ -0,0 +1,279 @@
+import { getConfigByNetwork } from '@deploy-configurations/configs'
+import { ADDRESS_ZERO } from '@deploy-configurations/constants'
+import {
+  ConfigEntry,
+  SystemConfig,
+  SystemConfigEntry,
+} from '@deploy-configurations/types/deployment-config'
+import { Network } from '@deploy-configurations/types/network'
+import { ServiceRegistry } from '@typechain/index'
+import { color } from 'console-log-colors'
+import { task } from 'hardhat/config'
+
+import {
+  getServiceRegistry,
+  isInvalidAddress,
+  ServiceRegistryMaybe,
+  VerificationResult,
+} from '../common'
+const { red, yellow, green } = color
+
+function getValidationStatusString(verificationResult: VerificationResult): string {
+  const verificationPercentage: number = Math.round(
+    (verificationResult.totalValidated / verificationResult.totalEntries) * 100,
+  )
+
+  if (verificationPercentage < 30) {
+    return (
+      red(`${verificationPercentage.toFixed(2)}%`) +
```

Comment:
It's not obvious what that the `red` function means. I prefer that notation:
```
import { log } from 'console-log-colors';

log.red('red text');
log.green('green text');
log.cyanBright('cyanBright text', 'cyanBright text2');
```

Reply:
The problem here is that I'm returning the colored strings, but not actually logging them, as I do that at the end of the calling function. I could alias those colors to say something like `red_text` or similar. What do you think?

Reply:
oh! okey, it's okey, I didn't catch that :)

Reply:
Added a comment to the declaration of those functions so it clarifies it a bit :)